README.md

```diff
@@ -140,9 +143,34 @@ Ein Codeblock, welcher bestimmte Abläufe vor und nach dem eigenen Code laufen l
 
 ### Common Environments
 
-* [itemize/enumerate](https://en.wikibooks.org/wiki/LaTeX/List_Structures)
-* [table/tabular](https://en.wikibooks.org/wiki/LaTeX/Tables)
-* [figure](https://en.wikibooks.org/wiki/LaTeX/Floats,_Figures_and_Captions)
+- [itemize/enumerate](https://en.wikibooks.org/wiki/LaTeX/List_Structures)
+- [table/tabular](https://en.wikibooks.org/wiki/LaTeX/Tables)
+- [figure](https://en.wikibooks.org/wiki/LaTeX/Floats,_Figures_and_Captions)
+
+## ZPMs Fontvorschlag
+
+> Die Standardfonts sind zwar ganz in Ordnung, aber ich finde, das geht schöner :^)
+
+Die vorgestellte Font-Konfiguration basiert auf [diesem Stackoverflow Thread](https://tex.stackexchange.com/a/114166), wo auch eine Vorschau betrachtet werden kann.
+**Die Fonts sind standardmäßig im Template aktiv.**
+
+Folgende Anpassungen sind in der ``build/components/package.config.tex`` zu finden:
+
+```latex
+\usepackage[scaled=0.88]{beraserif}
+\usepackage[scaled=0.85]{berasans}
+\usepackage[scaled=0.84]{beramono}
+\usepackage[T1]{fontenc}
+\usepackage{mathpazo}
+\usepackage[T1,small,euler-digits]{eulervm}
+\usepackage{listings} % Codeblöcke
+\lstset{
+  basicstyle=\ttfamily,
+    breaklines=true
+}
+```
```

Comment:
Ich glaub der code-block kann raus.
Muss eigentlich nicht nochmal separat in der readme stehen.